scripts/lib/CIME/test_scheduler.py

```diff
@@ -325,25 +325,25 @@ def _update_test_status(self, test, phase, status):
     def _shell_cmd_for_phase(self, test, cmd, phase, from_dir=None):
     ###########################################################################
         while True:
-            rc, output, errput = run_cmd(cmd, from_dir=from_dir)
+            rc, output, _ = run_cmd(cmd + " 2>&1", from_dir=from_dir)
```

Comment:
You can also do this with arg_stderr=subprocess.STDOUT,  it requires that you import subprocess but I think I prefer it to 2>&1